db/sqitch.conf

```diff
@@ -0,0 +1,8 @@
+[core]
```

Comment:
Oh wait, it's because this is in ``magprime`` and thus this file didn't exist until now.

Ignore me.